.github/workflows/pythonpackage.yml

```diff
@@ -22,6 +22,18 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
+    - name: Set up node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '14.x'
+    - name: Install rapidsnark
+      run: |
+        git clone https://github.com/nevermined-io/rapidsnark
+        cd rapidsnark
+        git submodule update --init --recursive
+        sh ./scripts/install-linux.sh
+        cd ..
+        rm -rf rapidsnark
```

Comment:
We need to make sure that this also works with `nevermined-tools` so I guess this dependencies will need to also be added to the dockerfile